datamuse/datamuse.py

```diff
@@ -0,0 +1,47 @@
+from urllib import urlencode
+import urllib2
+import json
+
+class Datamuse:
+    def __init__(self):
+        self.API_ENDPOINT = 'https://api.datamuse.com/words'
+
+    def make_api_request(self, params):
```

Comment:
Can you refactor this so `params` is assumed to be a dict and the `urlencode` is done here instead of by the caller?